### ISSUE TITLE
01_led.py parentheses

### ISSUE THREAD

**Issue by jgcrossman2** [sunfounder/Sunfounder_SuperKit_Python_code_for_RaspberryPi#14]
When I ran this from the command line, it worked. 

However when I went to change the time.sleep and ran it from the shell (Python 3.4.2 Shell), it threw up an error saying it was missing parentheses. I put parentheses around the '...led on' and 'led off...' and it worked on both.

I am new to Python, is this just related to the Shell that comes with Raspberry Pi?

**Comment by sunfounder** [comment]
No, the code in written in python 2, run it in python 3 will bring up bunch of errors. please try run it in python 2

**Comment by Dragonrun1** [comment]
As I pointed out in issue #9 all the scripts are now requesting python 3 so you might what to update your answer here.
